server/src/main/java/org/elasticsearch/search/aggregations/Aggregator.java

```diff
@@ -142,12 +142,12 @@ public BucketComparator bucketComparator(String key, SortOrder order) {
 
     /**
      * Build the results of this aggregation.
-     * @param owningBucketOrds the ordinals of the buckets that we want to
+     * @param ordsToCollect the ordinals of the buckets that we want to
      *        collect from this aggregation
      * @return the results for each ordinal, in the same order as the array
      *         of ordinals
      */
-    public abstract InternalAggregation[] buildAggregations(long[] owningBucketOrds) throws IOException;
```

Comment:
I *think* this is a more clear variable name. I made this change in a dead end when I was working on this change locally but I kind of like it.